src/commonMain/kotlin/character/classes/druid/talents/BalanceOfPower.kt

```diff
@@ -0,0 +1,32 @@
+package character.classes.druid.talents
+
+import character.Buff
+import character.Stats
+import character.Talent
+import mechanics.Rating
+import sim.SimParticipant
+
+/**
+ *
+ */
+class BalanceOfPower(currentRank: Int) : Talent(currentRank) {
+    companion object {
+        const val name = "Balance of Power"
+    }
+
+    override val name: String = Companion.name
+    override val maxRank: Int = 2
+
+    val buff = object : Buff() {
+        override val name: String = MoonkinForm.name
+        override val durationMs: Int = -1
+        override val hidden: Boolean = true
+
+        override fun modifyStats(sp: SimParticipant): Stats {
+            // TODO: is it easier to do this than add the 2% hit on each spell call?
```

Comment:
I wasn't sure how best to handle these types of things. It felt easier to just update the sims hit rating rather than do additional calculations per spell to add the additional hit percent. I was also trying to prevent confusion between using percents as 0 < pct < 1 and 0 < pct < 100

Reply:
It depends on the ability - many talents like this are spell-specific, so they would have to be limited.  Since Balance of Power applies to all spells, implementing as generic hit rating is just fine!